tests/phpunit/CRM/MembershipExtras/Job/OfflineAutoRenewal/SingleInstallmentPlanTest.php

```diff
@@ -1153,4 +1132,98 @@ private function assertPaymentPlanStructureIsOk($paymentPlanID, $expectedValues)
     );
   }
 
+  public function testDatesOfRenewalOfFixedMemberships() {
+    $startDate = '2019-07-01';
+    $endDate = '2019-11-30';
+    $receiveDateAfterRenewal = '2019-12-01';
+    $endDateAfterRenewal = '2020-11-30';
+
+    $fixedMembershipType = $this->createMembershipType([
+      'name' => 'Test Rolling Membership',
+      'period_type' => 'fixed',
+      'minimum_fee' => 0,
+      'duration_unit' => 'year',
+      'duration_interval' => 1,
+      'fixed_period_start_day' => '1201',
+      'fixed_period_rollover_day' => '1130',
+    ]);
+    $paymentPlanMembershipOrder = new PaymentPlanMembershipOrder();
+    $paymentPlanMembershipOrder->membershipStartDate = $startDate;
+    $paymentPlanMembershipOrder->paymentPlanFrequency = 'Yearly';
+    $paymentPlanMembershipOrder->paymentPlanStatus = 'Completed';
+    $paymentPlanMembershipOrder->lineItems = [
+      [
+        'entity_table' => 'civicrm_membership',
+        'price_field_id' => $fixedMembershipType->priceFieldValue['price_field_id'],
+        'price_field_value_id' => $fixedMembershipType->priceFieldValue['id'],
+        'label' => $fixedMembershipType->membershipType['name'],
+        'qty' => 1,
+        'unit_price' => $fixedMembershipType->priceFieldValue['amount'],
+        'line_total' => $fixedMembershipType->priceFieldValue['amount'],
+        'financial_type_id' => 'Member Dues',
+        'non_deductible_amount' => 0,
+        'auto_renew' => 1,
+      ],
+    ];
+    $paymentPlan = PaymentPlanOrderFabricator::fabricate($paymentPlanMembershipOrder);
+
+    $contributions = $this->getPaymentPlanContributions($paymentPlan['id']);
+    $this->assertEquals(1, count($contributions));
```

Comment:
there is no need for these two assertions, you are not testing if getPaymentPlanContributions work or not  at the end of the day